app/controllers/assignments_controller.rb

```diff
@@ -5,19 +5,18 @@ def index
   end
 
   def create
-    run Assignments::Create.with(client: panoptes_application_client),
```

Comment:
The app client was in there because regular Janes can't just create workflows in WG. What changed there?

Reply:
I didn't realize that the purpose was to override the app controller's version of client to do something that shouldn't normally be allowed by panoptes. 

Unsurprisingly, changing that back causes the op to throw a new error (`{"errors"=>[{"status"=>422, "source"=>{"pointer"=>"/data/attributes/client"}, "detail"=>"is not a valid object"}]}`) which I am debugging. It's been like this constantly since the rails 5 upgrade, with minor changes causing ActiveInteraction to complain in a variety of infuriating ways. This is why I removed the ones above. If you know what's up feel free to jump in, cause I've spent an embarrassing and unfortunate amount of time fighting with AI.